### DISCUSSION TITLE
thanhle.blog/may-man-co-hoi-va-su-chuan-bi

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#92] [Ideas]
# thanhle.blog/may-man-co-hoi-va-su-chuan-bi

https://thanhle.blog/blog/may-man-co-hoi-va-su-chuan-bi

**Comment by thanhlmm** [comment]
Trái ngược với may mắn - không may, sui rủi.
Chuyện xui rủi đâu muốn! Dối với mình thì may mắn và sui rủi khác nhau ở short term, nhưng về long term thì giống nhau. Xui thì vừa gặp đã khiến mình down rồi, nhưng vượt qua rồi đôi khi cũng thú vị, cũng có giá trị đấy chứ.
Sau này mình luôn biết ơn vì hồi nhỏ đã sống nghèo, đương nhiên hồi đó thì coi nhèo là một thứ mặc cảm lắm. Nhờ đã từng sống nghèo nên mình chịu khổ được, mình biết chắt chiu hơn, biết về quản lý tài chính. Đương nhiên nếu được làm lại, chả ai muốn nghèo đâu

**Comment by hai123vn** [comment]
Sáng nào đi làm, việc đầu tiên vào là em check mail, check blog của anh nhưng hôm qua lại không chắc tại vì mấy ngày trước có check thì không có bài mới hôm này vào check thì có bài mới ^^. Chúc anh 1 ngày tốt lành. Chắc bữa giờ bận quá không ra bài nhiều hở anh.

**Reply by thanhlmm** [reply]
lâu lâu bất ngờ xíu em :>

**Comment by ngtrieuvi92** [comment]
Bài hay đó em :D, nhưng chuyện may mắn & cơ hội này, thường thì phải trải qua vài lần learn & trả học phí khá đắt (vd 600tr về 20tr như trong bài) thì mới hiểu được.

**Comment by ntdat104** [comment]
Lại giống em future từ 10tr -> 40tr rồi rỗng túi :D
